packages/workspace/src/browser/workspace-service.ts

```diff
@@ -111,6 +113,11 @@ export class WorkspaceService implements FrontendApplicationContribution {
                 this.refreshRootWatchers();
             }
         });
+
+        const autoTrust = FrontendApplicationConfigProvider.get()['autoTrust'];
+        if (typeof autoTrust === 'boolean') {
+            this.trustRequest.resolve(autoTrust);
+        }
```

Comment:
I know the headline says 'light-weight,' but I'd prefer to also implement some of the [preferences](https://code.visualstudio.com/docs/editor/workspace-trust#_settings) associated with the feature so that users have some control.

Reply:
Good idea! I have implemented some of the settings in a follow-up change and rebased my branch.